ip/index.ts

```diff
@@ -618,13 +626,13 @@ function findIP(
 
   if (platform === "cloudflare") {
```

Comment:
Do we want to add a [Vercel platform check](https://vercel.com/docs/projects/environment-variables/system-environment-variables#VERCEL) because we know they set [`x-forwarded-for`](https://vercel.com/docs/edge-network/headers/request-headers#x-forwarded-for)?

Reply:
We need to support `x-forwarded-for` and `x-real-ip` for nginx and other proxies so those are handled by generic checks; however, that presents the question: if we can detect the platform, should we return an empty string if we can't detect the global IP from the platform-specific header? I think we should, which would mean we need to do an additional check for those 2 headers for vercel.

Reply:
> should we return an empty string if we can't detect the global IP from the platform-specific header?

I think so. It _should_ always be there, but we can fall back to our other logic if it isn't for whatever reason.

Reply:
I just opened https://github.com/arcjet/arcjet-js/pull/2021. Let's land that and then I'll add the Vercel checks in another PR to have a changelog entry.

Reply:
Followed up on that with #2022 and then I'll rebase this.